dialects/snowflake.yaml

```diff
@@ -371,30 +404,13 @@ aggregate_functions:
   aggregate: true
   supported_kernels:
   - bool
-- name: arithmetic_decimal.bitwise_and
-  local_name: bitand
-  supported_kernels:
-    - dec_dec
-- name: arithmetic_decimal.bitwise_or
-  local_name: bitor
-  supported_kernels:
-    - dec_dec
-- name: arithmetic_decimal.bitwise_xor
-  local_name: bitxor
-  supported_kernels:
-    - dec_dec
-- name: arithmetic_decimal.power
-  local_name: pow
-  required_options:
-    overflow: ERROR
-    complex_number_result: ERROR
-  supported_kernels:
-  - dec_dec
-- name: arithmetic_decimal.sqrt
-  local_name: sqrt
+- name: aggdec.count
+  local_name: COUNT
+  aggregate: true
   supported_kernels:
-    - dec
-- name: arithmetic_decimal.factorial
-  local_name: factorial
+  - any
+- name: aggdec.count
+  local_name: COUNT
+  aggregate: true
   supported_kernels:
-    - dec
```

Comment:
this should be dec

Reply:
is this count_star ? how is it in other dialects?

Reply:
Yes, this is count_star.  Although local dialect has one more entry(to be removed)

Other dialects have something like below. 

```
- name: aggregate_generic.count_star
  local_name: count(*)
  aggregate: true
  supported_kernels:
  - ''


```